llvm/test/CodeGen/RISCV/rvv/vmadd-vp.ll

```diff
@@ -476,8 +476,10 @@ define <vscale x 64 x i8> @vmadd_vv_nxv64i8(<vscale x 64 x i8> %a, <vscale x 64
 ; CHECK-LABEL: vmadd_vv_nxv64i8:
 ; CHECK:       # %bb.0:
 ; CHECK-NEXT:    vl8r.v v24, (a0)
-; CHECK-NEXT:    vsetvli zero, a1, e8, m8, tu, mu
-; CHECK-NEXT:    vmadd.vv v8, v16, v24, v0.t
+; CHECK-NEXT:    vsetvli zero, a1, e8, m8, ta, ma
+; CHECK-NEXT:    vmacc.vv v24, v8, v16
+; CHECK-NEXT:    vsetvli zero, zero, e8, m8, tu, ma
```

Comment:
I think we also need to look through FalseReg too so that we can still commute these cases, https://github.com/llvm/llvm-project/commit/1e826b2faae2c057ea21833fcba7fb27fa733523 should fix this